feat(ixp): document runtime extract in skill (#1480)
UiPath/skills: skills/uipath-ixp/SKILL.md

- User asks to label, review, or confirm document predictions
- User asks to improve extraction scores, prompts, or field instructions
- User asks to publish or manage IXP model versions

> - User asks to run a published/deployed model on a document — runtime extraction (async: `uip ixp start-extraction` → poll `uip ixp get-extraction-result`)

Reviewer: why not
```suggestion
- User asks to extract data from a document using a published/deployed model — runtime extraction
```

why is the command needed here? isnt the cli-reference explicit enough? (question for claude)

| "Describe this project" / "What's in it?" | Three calls, reported in order: (1) identity — `Title`/`Name` from `uip ixp projects get <project-name> --output json`; (2) current model — live/published + `TrainedTime` from `list-models`; (3) taxonomy — label-group/field counts from `uip ixp projects get-taxonomy <project-name> --output json` (raw artifact: counts live under `Data.dataset.label_groups` and `Data.dataset.entity_defs`, snake_case). Fold in performance (above) only if asked. Do NOT page `documents list` (its `Data` is a paged `{ Documents, Total, Offset, Limit }` envelope — use `Total` for a count) or read deployment bindings. Answer from these calls only (Critical Rule #1). |

> | "Run extraction on a document" / "Extract fields from this file with the published model" | **Async, two-step, keyed by project `Id` (GUID), not `Name`.** (1) Get the `Id`: if the user gave the GUID (or it's already in context from an earlier `projects list`), use it directly; otherwise resolve it via `uip ixp projects list -l 10000 --output json` → the entry whose `Title`/`Name` matches (ask if >1 match). (2) `uip ixp start-extraction <project-id> <file> --tag <tag> --output json` → capture `operationId` and persist it under `/tmp/ixp/<project-name>/extractions/`. (3) Poll `uip ixp get-extraction-result <project-id> <operation-id> --tag <tag> --output json` **every 5 s** while `Data.status` is `NotStarted`/`Running`; stop when `Data` is the fields array (done) or `Result: Failure`. No fixed time cap — extraction runs server-side regardless of polling, so quitting early neither cancels nor helps; if your own turns run out first, report the resumable `operationId`. `--tag` (`live` default) required on both; no `--version`. Full flow + polling: [CLI Reference § Runtime extraction](references/cli-reference.md#runtime-extraction). Distinct from `labellings get-predictions` (training set). |

Reviewer: the data here is duplicated with what we have described in the cli-reference. This should be shorter, in sync with the other action above that just call a cli command, only that it should describe it has two steps.
The Describe this project (just an example) is more complex and it give guidance not found in the cli-reference, that's why it's so verbose.

| "Describe this project" / "What's in it?" | Three calls, reported in order: (1) identity — `Title`/`Name` from `uip ixp projects get <project-name> --output json`; (2) current model — live/published + `TrainedTime` from `list-models`; (3) taxonomy — label-group/field counts from `uip ixp projects get-taxonomy <project-name> --output json` (raw artifact: counts live under `Data.dataset.label_groups` and `Data.dataset.entity_defs`, snake_case). Fold in performance (above) only if asked. Do NOT page `documents list` (its `Data` is a paged `{ Documents, Total, Offset, Limit }` envelope — use `Total` for a count) or read deployment bindings. Answer from these calls only (Critical Rule #1). |

> | "Run extraction on a document" / "Extract fields from this file with the published model" | Async two-step flow, keyed by project **`Id`** (GUID, not `Name`): `uip ixp extraction start <project-id> <file> --tag <live\|staging> --output json` → capture the `operationId`, then poll `uip ixp extraction get-result <project-id> <operation-id> --tag <tag> --output json` until it returns the fields or a failure. Resolve the `Id` via `projects list` if you don't already hold it. Full flow, polling, and file persistence: [CLI Reference § Runtime extraction](references/cli-reference.md#runtime-extraction). |

Reviewer: > Resolve the `Id` via `projects list` 

the project id

| "Upload these files" / "add documents" | **Project named / already in context →** supported; upload it (see the "Upload a document" row in Task Navigation). **Otherwise →** "Name an existing project and I'll upload it — or upload in-product (e.g. for a new project): https://docs.uipath.com/ixp/automation-cloud/latest/user-guide/building-and-deploying-models." |
| "Deploy this model" / "push to staging / production / folder / environment / tenant" | "I don't drive deployment to folders, environments, or tenants — that's a product-side flow: https://docs.uipath.com/ixp/automation-cloud/latest/user-guide/building-and-deploying-models. I can publish a model version with `uip ixp projects publish <project-name> --output json`, but binding it to a folder/environment is done in-product." |
| "Give X access" / "share this project" / "change roles or permissions" | "Access, roles, and permissions are managed in-product, not through this skill: https://docs.uipath.com/ixp/automation-cloud/latest/overview/managing-access." |

> | "Use this model in my automation / workflow / agent" / "wire the extractor into a process" | **Runtime extraction IS supported** — resolve the project `Id`, then `uip ixp start-extraction <project-id> <file> --tag <tag>` and poll `get-extraction-result` (see the "Run extraction on a document" row in Task Navigation). **Wiring** the model into an automation/workflow/agent as a node is authoring outside this skill: "Embedding a published model inside an automation is an authoring task outside this skill. See https://docs.uipath.com/ixp/automation-cloud/latest/user-guide/building-and-consuming-a-workflow." |

Reviewer: > **Runtime extraction IS supported**

this is relevant relative to the history of the file content, but it's not useful for claude's context

> | "Use this model in my automation / workflow / agent" / "wire the extractor into a process" | **Runtime extraction IS supported** — resolve the project `Id`, then `uip ixp start-extraction <project-id> <file> --tag <tag>` and poll `get-extraction-result` (see the "Run extraction on a document" row in Task Navigation). **Wiring** the model into an automation/workflow/agent as a node is authoring outside this skill: "Embedding a published model inside an automation is an authoring task outside this skill. See https://docs.uipath.com/ixp/automation-cloud/latest/user-guide/building-and-consuming-a-workflow." |

Reviewer: this is found under the ` Unsupported Capabilities` 😄 not the case anymore
